tests/functional/centos/7/docker-cluster/group_vars/all

```diff
@@ -0,0 +1,25 @@
+---
+
+ceph_stable: True
+mon_containerized_deployment: True
+osd_containerized_deployment: True
+mds_containerized_deployment: True
+rgw_containerized_deployment: True
+ceph_mon_docker_interface: eth1
+ceph_mon_docker_subnet: "192.168.15.0/24"
+ceph_osd_docker_extra_env: "CEPH_DAEMON=OSD_CEPH_DISK_ACTIVATE,OSD_JOURNAL_SIZE=100"
+ceph_docker_on_openstack: False
+public_network: "192.168.15.0/24"
+cluster_network: "192.168.16.0/24"
+journal_size: 100
+journal_collocation: true
```

Comment:
I had to add this and ``devices`` because of the mandatory variable check here https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-osd/tasks/main.yml#L2

Do we want to not include that file when ``osd_containerized_deployment: True`` instead?